jsonschema2pojo-cli/src/main/scripts/jsonschema2pojo.bat

```diff
@@ -1,18 +1,18 @@
-@REM
-@REM Copyright © 2010-2014 Nokia
-@REM
-@REM Licensed under the Apache License, Version 2.0 (the "License");
-@REM you may not use this file except in compliance with the License.
-@REM You may obtain a copy of the License at
-@REM
-@REM      http://www.apache.org/licenses/LICENSE-2.0
-@REM
-@REM Unless required by applicable law or agreed to in writing, software
-@REM distributed under the License is distributed on an "AS IS" BASIS,
-@REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-@REM See the License for the specific language governing permissions and
-@REM limitations under the License.
-@REM
-
-@echo off
-java -jar "%~dp0/../lib/${project.build.finalName}.jar" %*
+@REM
+@REM Copyright © 2010-2014 Nokia
+@REM
+@REM Licensed under the Apache License, Version 2.0 (the "License");
+@REM you may not use this file except in compliance with the License.
+@REM You may obtain a copy of the License at
+@REM
+@REM      http://www.apache.org/licenses/LICENSE-2.0
+@REM
+@REM Unless required by applicable law or agreed to in writing, software
+@REM distributed under the License is distributed on an "AS IS" BASIS,
+@REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+@REM See the License for the specific language governing permissions and
+@REM limitations under the License.
+@REM
+
+@echo off
+java -jar "%~dp0/../lib/${project.build.finalName}.jar" %*
```

Comment:
Your branch is still showing a change in this file from CRLF line endings to LF. Could you maybe `git checkout origin/master jsonschema2pojo-cli/src/main/scripts/jsonschema2pojo.bat` and commit that?

Reply:
I have added .bat to the .gitattributes file and renormlized that bat file. It may still show changes because it was original in the git repo as LF.

https://stackoverflow.com/a/47580886/45708

Reply:
Hi Dan. Everything looks good but this is the last problem in the PR. In `master` this file uses CRLF (Windows) line endings not LF:

```
$ file jsonschema2pojo-cli/src/main/scripts/jsonschema2pojo.bat
jsonschema2pojo-cli/src/main/scripts/jsonschema2pojo.bat: DOS batch file, UTF-8 Unicode text, with CRLF line terminators
```

You can also try `vim -b jsonschema2pojo-cli/src/main/scripts/jsonschema2pojo.bat` to see the `^M` characters.

In your branch (this PR) changes the line endings in this file to LF.

I've actually changed my mind about this `.gitattributes` file, it's adding more confusion and I don't want git doing anything to change the files under version control. Please remove `.gitattributes` and also remove your changes to this file (`git checkout origin/master jsonschema2pojo-cli/src/main/scripts/jsonschema2pojo.bat`). Then we're good to go.

Reply:
Done.